src/jit/MemoryInl.h

```diff
@@ -730,6 +757,12 @@ static void emitStore(sljit_compiler* compiler, Instruction* instr)
             return;
         }
 
+#if (defined SLJIT_CONFIG_ARM_32 && SLJIT_CONFIG_ARM_32)
+        if (simdType == (SLJIT_SIMD_STORE | SLJIT_SIMD_REG_64 | SLJIT_SIMD_ELEM_8)) {
```

Comment:
Just a question: If the const stored in `simdType` is only used for comparison here, why don't just compare it to the original value: 

`SLJIT_SIMD_STORE | SLJIT_SIMD_REG_128 | SLJIT_SIMD_ELEM_64 | SLJIT_SIMD_FLOAT`

This way setting an arm32 specific value above is unnecessary. This could be done in other cases (if possible).